src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1427,9 +1428,10 @@ public function testInheritEnvDisabled()
      *
      * @return Process
      */
-    private function getProcess($commandline, $cwd = null, array $env = null, $input = null, $timeout = 60, array $options = array())
+    private function getProcess($commandline, $cwd = null, array $env = null, $input = null, $timeout = 60)
     {
-        $process = new Process($commandline, $cwd, $env, $input, $timeout, $options);
+        $process = new Process($commandline, $cwd, $env, $input, $timeout);
+        $process->inheritEnvironmentVariables(true, false);
```

Comment:
Unfortunately no - so yes, this call is required.
Other comments addressed also.